lms/djangoapps/organizations/views.py

```diff
@@ -175,3 +184,27 @@ def groups(self, request, pk):
             organization.groups.add(group)
             organization.save()
             return Response({}, status=status.HTTP_201_CREATED)
+
+    @action(methods=['get', ])
+    def user_ids(self, request, pk):
```

Comment:
I feel like these action should be 'users', 'groups', and 'workgroups', and we should control the resulting response (ie, the representation) payload using query parameters.  We've done some work around this in the main edx-platform APIs using the concept of "expanded references" (see https://openedx.atlassian.net/wiki/display/AC/edX+REST+API+Conventions#edXRESTAPIConventions-#expandExpandingreferencedobjects).  In this case the default response for '/users' could be the id list, and if additional information is required the client could specify something like "expand=true".